src/index.html

```diff
@@ -16,17 +16,33 @@
       href="style.css"
     />
   </head>
-  <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+  <body class="search-bar__page">
```

Comment:
Use semantic tags where possible. Instead of using a div with a class of 'search-bar__page', consider using a semantic tag like 'main' or 'section' to wrap the forms.